common/lib/capa/capa/templates/choicegroup.html

```diff
@@ -1,14 +1,16 @@
 <form class="choicegroup capa_inputtype" id="${id}-form">
-    <fieldset role="${input_type}group" aria-label="${label}">
+    <fieldset>
       % if response_data['label']:
         <legend id="${id}-legend" class="response-fieldset-legend field-group-hd">${response_data['label']}</legend>
+      % else:
+        <legend>Question</legend>
```

Comment:
@lamagnifica, I wanted to make you aware of this since we will probably want to cover it in Docs.  Since it would be very bad to have a fieldset without a legend (the legend serves as the accessible name for a set of radio buttons or checkboxes) we decided to just put the word "Question" as the legend value if an author authors a question in such a way where there is no legend value.  In the Docs, we may want to have something in there that says "Hey, if you are seeing the word Question above your checkboxes, you should probably add some actual question text, ya big dummy"  :)